src/pages/Checkout/CheckoutPage.vue

```diff
@@ -212,6 +212,13 @@
 				@credit-removed="handleCreditRemoved"
 				@promo-opt-out-lightbox-closed="handleCancelPromoOptOut"
 			/>
+			<campaign-join-team-form
+				v-if="this.showTeamForm"
+				:campaign-name="campaignPartnerName"
+				:team-id="this.teamId"
+				:promo-id="this.promoFundId"
+				@team-process-complete="handleTeamJoinProcess"
```

Comment:
@roger-in-kiva Need to remove `this` on all of these fields.

Reply:
I see that's how it's setup in the CCLandingPage too. But `this` shouldn't be necessary...

Reply:
Agreed!